rubygem-patternfly-sass/rubygem-patternfly-sass.spec

```diff
@@ -0,0 +1,74 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name patternfly-sass
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 2.8.0
+Release: 1%{?dist}
+Summary: Red Hat's Patternfly, converted to Sass and ready to drop into Rails
+Group: Development/Languages
+License: ASL 2.0
+URL: https://github.com/Patternfly/patternfly-sass
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: %{?scl_prefix_ror}rubygem(bootstrap-sass) >= 3.3.5
+Requires: %{?scl_prefix_ror}rubygem(bootstrap-sass) < 3.4.0
+Requires: %{?scl_prefix_ror}rubygem(font-awesome-sass) >= 4.3.0
+Requires: %{?scl_prefix_ror}rubygem(font-awesome-sass) < 4.4.0
```

Comment:
no _ror on these bootstrap/font-awesome dependencies, as you're adding them into our SCL collection